packages/react-cosmos-playground/cosmos.config.js

```diff
@@ -2,4 +2,5 @@ module.exports = {
   globalImports: ['src/utils/global.less'],
   exclude: /_shared/,
   port: 8090
+  // responsiveDevices: [{ label: 'Max', width: 1000, height: 500 }]
```

Comment:
⬆️

Reply:
Thanks, have removed. What do you think of that API btw for allowing users to configure their own viewport defaults that show up in the responsive menu/header?

I'd like to also allow users to define the viewport in the fixture using the `label` rather than having to memorize dimensions, e.g.:

```
component: MyComponent,
props: {},
viewport: "iPhone 5"
```